src/quo2/components/dividers/date.cljs

```diff
@@ -0,0 +1,20 @@
+(ns quo2.components.dividers.date
+  (:require [react-native.core :as rn]
+            [clojure.string :as string]
+            [quo2.components.markdown.text :as quo2.text]
```

Comment:
usually we don't use quo2 in alias, just text and colors

Reply:
Updated